session_2/student_registry/app/models/student.rb

```diff
@@ -0,0 +1,3 @@
+class Student < ApplicationRecord
+    validates :cgpa, numericality: {only_integer: true, less_than_or_equal_to: 10.0, greater_than_or_equal_to: 0.0}
```

Comment:
Wait, doesn't `only_integer: true` seem wrong to you - Can cgpa be only integer values?

Reply:
I will fix that.